postgres/src/model.py

```diff
@@ -0,0 +1,12 @@
+from typing import List
+
+from pydantic import BaseModel
+
+
+class Task(BaseModel):
+    id: int
+    task: str
```

Comment:
enhancement: You could also have a `timestamp`, `task_completion` and `status` of a task for bit more elaborate task.